src/sentry/models/notificationaction.py

```diff
@@ -146,10 +148,12 @@ class NotificationAction(AbstractNotificationAction):
     _trigger_types: List[Tuple[int, str]] = ActionTriggerType.as_choices()
 
     organization = FlexibleForeignKey("sentry.Organization")
-    project = FlexibleForeignKey("sentry.Project", null=True)
+    projects = models.ManyToManyField("sentry.Project")
     # TODO(Leander): After adding AlertRuleTriggerAction to HybridCloudForeignKey, we can remove these lines
     integration = None
     integration_id = HybridCloudForeignKey("sentry.Integration", on_delete="CASCADE")
+    sentry_app = None
```

Comment:
```suggestion
```

Reply:
@markstory I actually have to retain that so the `sentry_app` field that's present on the abstract field doesn't apply to this new model. 

Whenever we break the FKs on `integration` and `sentry_app` for `AlertRuleTriggerAction`, all these lines can be safely removed in favour of keeping them in the abstract model.